packages/apollo-server-fastify/README.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Fastify
+description: Setting up Apollo Server with Fastify
+---
+
+[![Coverage Status](https://coveralls.io/repos/github/apollographql/apollo-server/badge.svg?branch=master)](https://coveralls.io/github/apollographql/apollo-server?branch=master) [![Get on Slack](https://img.shields.io/badge/slack-join-orange.svg)](https://www.apollographql.com/#slack)
+
+This is the Fastify integration of GraphQL Server. Apollo Server is a community-maintained open-source GraphQL server that works with all Node.js HTTP server frameworks: Express, Connect, Fastify, Hapi, Koa and Restify. [Read the docs](https://www.apollographql.com/docs/apollo-server/). [Read the CHANGELOG.](https://github.com/apollographql/apollo-server/blob/master/CHANGELOG.md)
+
+```sh
+npm install apollo-server-fastify
+```
+
+## Fastify
+
+```js
+import fastify from 'fastify';
+import jsonParser from 'fast-json-body';
+import { graphqlFastify } from 'apollo-server-fastify';
+
+const myGraphQLSchema = // ... define or import your schema here!
+const PORT = 3000;
+
+const app = fastify();
+
+// jsonParser is needed for POST.
+app.addContentTypeParser('application/json', function(req, done) {
```

Comment:
Hi! I'm one of the Fastify maintainers :)
Thank you for implementing this!
Just a nit, you don't need to add a content type parse for `application/json`, since it's included directly in Fastify!
Besides Fastify performs many security checks that `fast-json-body` is not doing.

Reply:
@delvedor Thanks a lot for the pointer :) But I implemented this 4 months back and for some reason I had to explicitly add the content parser. But I am working on V2 of this plugin and will take your point into consideration while doing that